src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ env:
   MAIN_PYTHON_VERSION: "3.8"
   DOCUMENTATION_CNAME: '{{ cookiecutter.__solution_name_slug }}.docs.solutions.ansys.com'
   # When changing this value, update setup_environment.py --build-system-version default value to match
-  POETRY_VERSION: "1.4"
+  POETRY_VERSION: "1.4.2"
   TOX_VERSION: "3.27.1"
```

Comment:
``tox`` version is no longer necessary with the update of the actions.

Reply:
Please upgrade the reusable action as well.

Reply:
@iazehaf done